documentation/wiki/Building-Testing-and-Debugging-on-Full-Framework-MSBuild.md

```diff
@@ -25,6 +25,10 @@ To run the unit tests from Visual Studio:
 
 To build MSBuild and run all unit tests from the command line, use `.\build.cmd -test`.
 
+Some tests are creating symlinks to test associated functionality - in order for them to succeed you have two options:
+* Run those tests elevated
```

Comment:
Is there still a test that fails if you run elevated then run not-elevated? If so, might want to mention that and how to fix it.

Reply:
Creating symlinks require elevation or developer mode on each run.

Would you suggest different wording informing about this behavior?

Reply:
I'd put the option to enable developer mode first, and I'd add a warning for running tests elevated that if you then run not elevated, you have to do the following steps (and list out any tests that fail with running elevated --> not elevated along with an explanation of how to fix it—I think there was just one, and it could be fixed by deleting some directory...or you could fix the test to clean up properly.)

Reply:
Switched the order of options.

I'm not aware of issues with tests when switching elevation mode. The newly added test - `BinaryLoggerShouldEmbedSymlinkFilesViaTaskOutput` - uses transient files. The pre-existing test that uses symlinks - [`DoNotFollowRecursiveSymlinks`](https://github.com/dotnet/msbuild/blob/main/src/Shared/UnitTests/FileMatcher_Tests.cs#L72) cleanup after itself as well.
Am I missing any other case?
If we have any - I'd definitely prefer to do a proper cleanup in tests rather than scare reader by extra steps in wiki. Please point me :-)

Reply:
@Forgind I've run all tests elevated and then without elevation and I do not observe any issues. Did you have any specific test or code issue in mind?
If not - do you feel there is any further improvement that can be made to the touched code or doc?

Reply:
TestCache2 was the one I had in mind that had failed recently due to admin --> not admin running of the test. I believe it specifically failed if a particular folder did not exist prior to the test's execution, then you ran the test as an admin, then you ran the test as a normal user. It's possible that's been fixed since that came up, though; it was a while ago.

Reply:
This seems to be addressed here: https://github.com/dotnet/msbuild/pull/8122

Do you feel there are any further improvement that can be made in scope of this PR?

Reply:
Yup; I forgot about that. That's all I can think of.